src/components/MyAccount/ModalHeading.tsx

```diff
@@ -0,0 +1,21 @@
+import { Heading } from "@nypl/design-system-react-components"
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+// this method is a workaround because the size of the heading can't be updated
+// from the css module. it's a method because the headingText prop of the modal
+// props object requires ONLY a Heading component and won't accept a custom
+// component to be passed in.
```

Comment:
This is interesting. For the first part, I think it's because the style props or using the `sx` or `__css` have higher priority than the classname rules. So it sounds like the use of CSS modules isn't completely supported.

The `headingText` prop should accept a string or HTML element. We do prefer a DS `Heading` component but you should be able to pass any other element such as `h2`, `h3`, etc. or even `<span>`s. But you do lose the default styling if it's a span. Can you share what you tried that didn't work?

Reply:
replaced that method invocation with `<h5>`'s. thanks for the suggestion.